skins/oasis/js/Tracking.js

```diff
@@ -94,29 +87,44 @@ jQuery(function ($) {
 			}
 
 			switch (id) {
-			case 'comment':
-				label = el.hasClass('talk') ? 'talk' : 'comment';
-				break;
-			case 'edit':
-				label = id;
-				break;
-			case 'delete':
-			case 'history':
-			case 'move':
-			case 'protect':
-			case 'flags':
-				label = 'edit-' + id;
-				break;
+				case 'createpage':
+					label = 'add-a-page';
+					break;
+				case 'comment':
+					label = el.hasClass('talk') ? 'talk' : 'comment';
+					break;
+				case 'share':
```

Comment:
I would say that the entry point could be tracked with the rest of the `title-area`, but the icons themselves in the modal should be tracked in `social-share` category. We could ask Szeryf, as he'll look at the data.